CHANGELOG.md

```diff
@@ -7,6 +7,106 @@ players.
 
 ## [0.9]
 
+### [0.9.0]
+
+#### Release summary
+
+TODO - write info about the release comparing with 0.8.2.
```

Comment:
I'm wondering if this would be enough...
```suggestion
This release is primarily focused on offering support for the latest JupyterHub (version 1.1.0) and the latest versions of the JupyterHub authenticators. There are also additional configuration options and a number of bug fixes.
```